### DISCUSSION TITLE
Can you consider supporting the parsing of CUE files?

### DISCUSSION THREAD

**Discussion by mapleshadow** [navidrome/navidrome#2813] [Ideas]
Many music files are single-track lossless FLAC files, but they all have CUE files. Can you consider supporting them? Thank you to the development team!

**Comment by metalheim** [comment]
@PSyton already implemented CUE Sheet parsing here: https://github.com/navidrome/navidrome/pull/2201
Reading the latest comment, he needs time to polish it and iron out some bugs with Wavpack files, but otherwise its pretty much done

**Reply by mapleshadow** [reply]
Looking forward to the early implementation of this feature, thank you for your great efforts!
